GetDKAN/dkan
# Fix contextual filter on list_of_users_groups view

The user profile displays a list of groups a user belongs to, the contextual filter is set to use the logged in user rather than the user being viewed so the results are not accurate.

## How to reproduce
1. Create a user and add them to a group
2. View the new user's profile page and note that the list of groups on the left displays the groups of the user you are logged in as and not the user you are viewing.

## QA Steps

- [x]  Create a user and add them to a group
- [x]  View the new user's profile page and confirm that the list of groups on the left displays correct groups for the user being viewed.

## Reminders
